src/renderer/getDi.tsx

```diff
@@ -4,25 +4,25 @@
  */
 
 import { createContainer } from "@ogre-tools/injectable";
+import { autoRegister } from "@ogre-tools/injectable-extension-for-auto-registration";
+import { registerMobX } from "@ogre-tools/injectable-extension-for-mobx";
 import { Environments, setLegacyGlobalDiForExtensionApi } from "../extensions/as-legacy-globals-for-extension-api/legacy-global-di-for-extension-api";
 
 export const getDi = () => {
-  const di = createContainer(
-    getRequireContextForRendererCode,
-    getRequireContextForCommonExtensionCode,
-    getRequireContextForCommonCode,
-  );
+  const di = createContainer();
+
+  registerMobX(di);
```

Comment:
Can you explain what this does and why it is needed?

Reply:
This is extension for injectable which permits us to injectMany in a reactive way. No usages in this PR yet, but used e.g. to get rid of all `extensionInstallationCounter`

Reply:
Still, we removed this for being YAGNI.